### DISCUSSION TITLE
Book made by Posts

### DISCUSSION THREAD

**Discussion by arthurrio** [dailydotdev/daily#1125] [Feature Requests]
# 📚 Book made by posts

### Introduction
The idea came from I was thinking of building playlists using different posts, to do this I can pin a lot of posts, use tags, or even create a post with links to another post but how cool it be if, in the end, it looks like a "book"? 
I'm not talking about compiling all posts in one but finding posts around the feeds or creating posts to this and creating post lists about one subject that could be versioned, collaboratively built, and placed on profiles or in a squad area.

### Idea
This idea is not new for sure, could be like a reading list in that you can put a Title and organize your posts in chapters, and in each chapter could have posts made by me or posts that I found in the daily.dev that could be a useful resource for this **book/reading list/collection**

Could be a collaborative **book/reading list/collection** where we could use phases to control the versions like git, and in this way we could update this **book/reading list/collection** with new posts or remove old ones, the cool thing here is keeping the history to be trackable like a book when launching a new edition every year with something new or changing something.

In this way, I could build a **book/reading list/collection** with my squad about algorithms, where we could add posts about data structures, and code challenges, and organize topics by companies, these posts would be like a curation about the bests posts to learn something or some subject, could be written by me, by my squad or a collection of greats posts.

### Goals

- Be possible to have more than one bookmark
  - Define the name of bookmarks
- Create a special bookmark collection(Book)
  - Can have more than one bookmark or not
- A special section(bookshelf) for the special bookmarks collection
  - A section in the squad and profile  
- Collaborative - Build together by permission
- Version control
  - button to persist the actual version like a git's tag name 

## Images idea

![Book Index](https://github.com/dailydotdev/daily/assets/72942190/3a0513e1-09f7-434e-aa54-6caeb8118c5e)

![Chapter Example](https://github.com/dailydotdev/daily/assets/72942190/3dd3496f-9894-4263-8c6a-e8085f16adb7)

**Comment by idoshamun** [comment]
Hi @arthurrio,

Thanks for the effort and the well written feature request 🙏 
It's a lovely idea, I'll raise it internally but can't promise anything at this stage 😬

**Reply by arthurrio** [reply]
Thank you for your time reading my idea! 
For sure, there are a lot of features in one "Feature Request". I just tried to write before I forgot haha
If it's missing something, I'm able to add more details
